bin/fusor-update-repos

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import json
+import sys
+import getopt
+from subprocess import Popen, PIPE, STDOUT
+
+def usage(name):
+  print '\tUsage: %s -u <username> -p <password> -o <organiztion>\n' % name
+
+def main(argv):
+  found_u = False
+  found_p = False
+  found_o = False
+
+  try:
+    opts, args = getopt.getopt(argv,"hu:p:o:",["uname=","passwd=","org"])
+  except getopt.GetoptError:
+    usage(sys.argv[0])
+    sys.exit(1)
+  for opt, arg in opts:
+    if opt == '-h':
+      usage(sys.argv[0])
+      sys.exit()
+    elif opt in ("-u", "--uname"):
+      username = arg
+      found_u = True
+    elif opt in ("-p", "--passwd"):
+      password = arg
+      found_p = True
+    elif opt in ("-o", "--org"):
+      organization = arg
+      found_o = True
+  if not (found_u and found_p and found_o):
+      print '[ERROR] username, password, and organization options are required!'
+      usage(sys.argv[0])
+      sys.exit(2)
+
+  # get the list of repositories
+  cmd = "hammer -u %s -p %s --output='json' repository list --organization='%s'" % (username, password, organization)
+  data = Popen(cmd, shell=True, stdin=PIPE, stdout=PIPE, stderr=STDOUT, close_fds=True)
```

Comment:
You may want to consider adding an error check here like you did on line 51.

If an error occurs, you'd want to print out stdout/stderr to help debug.